miniapp/kernel/miniapp_larft_gemv.cpp

```diff
@@ -0,0 +1,230 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2024, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <future>
```

Comment:
Used?

Reply:
Yes. see kernel_runner.h

Reply:
Why is it not included in `kernel_runner.h` then? It's not used directly in this file, or am I missing it somewhere?

Reply:
I already moved there in the commit mentioned above.

Reply:
Ok, wrong commit... this is the correct one: 2caad6a1905387822bea49483c07f7eec78d6877

Reply:
I see, makes more sense now. Thanks!